client/coral-admin/src/containers/ModerationQueue/ModerationContainer.js

```diff
@@ -70,8 +70,10 @@ class ModerationContainer extends Component {
       <div>
         <ModerationHeader asset={asset} />
         <ModerationMenu
-          activeTab={activeTab}
           asset={asset}
+          premodCount={data.premod.length}
```

Comment:
+1, the front end isn't going to hold all of the things in the queue so we can't base the count on fe data.